llvm/lib/Target/AMDGPU/AMDGPUCallLowering.cpp

```diff
@@ -230,13 +230,6 @@ struct AMDGPUOutgoingArgHandler : public AMDGPUOutgoingValueHandler {
     return AddrReg.getReg(0);
   }
 
-  void assignValueToReg(Register ValVReg, Register PhysReg,
-                        const CCValAssign &VA) override {
-    MIB.addUse(PhysReg, RegState::Implicit);
-    Register ExtReg = extendRegisterMin32(*this, ValVReg, VA);
-    MIRBuilder.buildCopy(PhysReg, ExtReg);
-  }
```

Comment:
Not sure how you can delete this, or is related

Reply:
This is overriding the base class implementation `AMDGPUOutgoingValueHandler::assignValueToReg` and the only difference is that it _removes_ the readfirstlane stuff.